File: src/main/java/com/fish/entity/pojo/MessageType.java
package com.fish.entity.pojo;


import com.mybatisflex.annotation.Id;
import com.mybatisflex.annotation.KeyType;
import com.mybatisflex.annotation.Table;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 *  实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(value = "oj_message_type")
public class MessageType implements Serializable {

    @Id(keyType = KeyType.Auto)
    private Long typeId;

    private String typeName;

    private String typeDescribe;

}

File: src/main/kotlin/com/fish/entity/pojo/MessageType.kt
package com.fish.entity.pojo

import com.mybatisflex.annotation.Id
import com.mybatisflex.annotation.KeyType
import com.mybatisflex.annotation.Table
import java.io.Serializable

/**
 * 实体类。
 *
 * @author fish
 * @since 2023-11-14
 */
@Table(value = "oj_message_type")
open class MessageType(
    @Id(keyType = KeyType.Auto)
    var typeId: Long?,
    var typeName: String?,
    var typeDescribe: String?,
) : Serializable
